### PR TITLE
feat(profiling): Add index on profile_id

### PR DESCRIPTION
**Pull request by phacops**

### CODE REVIEW

**Comment by fpacifici** [comment]
@phacops  Could you please merge this ?

**Comment by lynnagara** [comment]
@phacops What's the status of this migration? Do we need this index?

File: CODEOWNERS [hunk 1]
```diff
@@ -13,3 +13,5 @@
 /cloudbuild.yaml                @getsentry/releases
 /scripts/bump-version.sh        @getsentry/releases
 /scripts/post-release.sh        @getsentry/releases
+
+/snuba/migrations/snuba_migrations/profiles @getsentry/profiling
```
Comment (lynnagara) [inline CODEOWNERS:17]: Mind removing this line? The path has changed so it is causing a failure. Though it is our eventual goal that different teams will own separate paths within the Snuba repo we are not quite there yet.

**Review by lynnagara — CHANGES_REQUESTED** [review]

File: snuba/migrations/groups.py [hunk 2]
```diff
@@ -230,7 +230,7 @@ def __init__(self) -> None:
         super().__init__("snuba.migrations.snuba_migrations.profiles")
 
     def get_migrations(self) -> Sequence[str]:
-        return ["0001_profiles"]
+        return ["0001_profiles", "0002_profiles_add_index_for_profile_id"]
```
Comment (lynnagara) [inline snuba/migrations/groups.py:233]: Please rebase, and make this migration 0004 and 0002 exists now.

**Comment by lynnagara** [comment]
This can't be merged currently since index on UUID is not supported in our minimum supported clickhouse version (20.3).

**Pull request closed without merging**